setup.cfg

```diff
@@ -0,0 +1,4 @@
+[flake8]
+exclude=invoke/vendor,sites,tests
```

Comment:
I force-pushed a change that now includes tests too. I just didn't want the diff to be too big :)